specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -6452,6 +6540,39 @@
           }
         }
       }
+    },
+	"CustomerPolicy": {
```

Comment:
>Customer [](start = 2, length = 8)

Is the customer in this case a user that exists in AAD? (Can standard Azure RBAC be used here instead of this?)

Reply:
Customer represents a customer relationship for partner led billing. We use MCapi authZ for everything.

Reply:
Makes sense.